include/yaml-cpp/parser.h

```diff
@@ -11,6 +11,8 @@
 #include <memory>
 
 #include "yaml-cpp/dll.h"
+#include "yaml-cpp/directives.h"
```

Comment:
Is the idea that these are missing includes? What errors or warnings are you getting? I don't see the word "unique_ptr" in the link you sent.

Reply:
wrong link https://ci.appveyor.com/project/Dekken/statick/builds/23102012